### ISSUE TITLE
Create a CLS temperature monitor control

### ISSUE THREAD

**Issue by helfrij** [acquaman/acquaman#2088]
Since the temperature monitors follow a very specific PV name structure, it's possible to have a general CLS temperature monitor control that accesses the temperature monitor status and value.

I imagine something like this could be really useful in conjunction with the utilities view on BioXAS. Right now, only the global temperature monitors status is shown, but I think the next iteration of the view could provide more detail: perhaps providing individual views only for temperature monitors that have a bad status.

The changes to the view would be taken care of in a separate issue, and likely after #2041. This issue would address creating a general temperature monitor control.

**Comment by helfrij** [comment]
Done in #2149.

Time Estimate: 4 hours.
